docs/reference/commandline/images.md

```diff
@@ -343,3 +344,39 @@ b6fa739cedf5        committ                   latest
 746b819f315e        postgres                  latest
 {% endraw %}
 ```
+
+## Customizing display orders
+
+To customize list orders, you can use `--sort-by` option, this option takes format
+of `field[:order]`. `field` should be a case insensitive string among "repository",
```

Comment:
s/should be/is

How about `field` is one or more of: `repository`, `tag`, `digest`, `createdat`, `id`, or `size`.

(I assume you can sort by more than one)